documentation/general/deployment.md

```diff
@@ -201,12 +204,12 @@ it. Please see the
      this yourself (i.e., if you do not have the ability to make changes to our
      Terraform configuration)
 
-2. Deploy the code that depends on the new or updated variable by dispatching
-   the appropriate deployment workflow for the application and environment that
-   was updated.
+2. Deploy the code that depends on the new or updated variable by publishing the
+   release for the application and environment that was updated.
    - If the application only needs to be re-deployed to the already
-     running-version, dispatch the workflow with the version present at the
-     version endpoint for the application. See the "version endpoint" in the
+     running-version, dispatch the appropriate deployment workflow in the
+     WordPress/openverse-infrastructure repository with the application's
+     current version. See the "version endpoint" in the
```

Comment:
Just curious why the docs here say to use the version from the version endpoint, rather than the rel-tag. Is there an intended difference?

Reply:
There is no underlying difference, they will point to exactly the same image, the rel- tags are aliases for the commit hash tags the version endpoint returns. From a process perspective, however, it is impossible to confuse which image tag to use if you follow the version endpoint, whereas finding the rel- tag is potentially fraught.

Reply:
Makes sense, thanks!